### PR TITLE
[NG] New SMI page

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR updates the SMI page.
https://deploy-preview-1337--layer5ng.netlify.app/projects/smi

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by chandrashritii** [comment]
Checkpoints - 

- Left Indent the first paragraph under the primary header
- Infuse Meshery into the content. Add the Meshery logo on the page somewhere. (Could be used as a bullet point)
- Proof read content

File: src/sections/SMI/data.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+const data = {
+    pricings: [
```
Comment (Tanuj22) [inline src/sections/SMI/data.js:3]: Better name

File: src/sections/SMI/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                            <h6> Why does conformance matter? </h6>
+                            <h1> Service Mesh Interface Conformance</h1>
+                            <h5 className="desc-p"> 
+                            In order to confidently operate a service mesh without locking into the specific service mesh's APIs, you will adopt SMI. How do you know if the service mesh you are using is SMI compatible, though
+                            </h5>
+                            <Button primary title="Run SMI Conformance" /> 
+                        </Col>
+                        <Col lg={4} md={12} sm={12}>
+                            <div className="hero-img">
+                                <img src={smi}></img>
```
Comment (Tanuj22) [inline src/sections/SMI/index.js:31]: Add `alt`

File: src/sections/SMI/index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    return (
+        <Layout>
+            <SMIWrapper>
+                <Container>
+                    <img className="logo" src={logo} />
+                    <Row className="description">
+                        <Col className="desc-text" lg={8} md={12} sm={12}>
+                            <h6> Why does conformance matter? </h6>
+                            <h1> Service Mesh Interface Conformance</h1>
+                            <h5 className="desc-p">
```
Comment (Tanuj22) [inline src/sections/SMI/index.js:24]: This is a paragraph not a heading

File: src/sections/SMI/index.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                                sample application being fitted to each service mesh.
+                                </p>
+                            </Col>
+                            <Col lg={4}>
+                                <Button primary title="Run SMI Conformance"></Button>
+                            </Col>
+                        </Row>
+                    </div>
+                </div>
+                {/* </Container> */}
```
Comment (Tanuj22) [inline src/sections/SMI/index.js:85]: Remove

File: src/sections/SMI/index.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                <Container>
+                    <img className="logo" src={logo} />
+                    <Row className="description">
+                        <Col className="desc-text" lg={8} md={12} sm={12}>
+                            <h6> Why does conformance matter? </h6>
+                            <h1> Service Mesh Interface Conformance</h1>
+                            <h5 className="desc-p"> 
+                            In order to confidently operate a service mesh without locking into the specific service mesh's APIs, you will adopt SMI. How do you know if the service mesh you are using is SMI compatible, though
+                            </h5>
+                            <Button primary title="Run SMI Conformance" />
```
Comment (Tanuj22) [inline src/sections/SMI/index.js:27]: what is the link for this button?

File: src/sections/SMI/smi.style.js [hunk 6]
```diff
@@ -0,0 +1,146 @@
+import styled from "styled-components";
+
+const SMIWrapper = styled.section`
+    .description{
+        margin: 6.25rem -0.94rem 7.5rem -0.94rem;
+        h6{
+            font-size: 20px;
```
Comment (Tanuj22) [inline src/sections/SMI/smi.style.js:7]: Why add custom font size. We already have sizes defined globally

File: src/sections/SMI/smi.style.js [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+            color: #1E2117;
+            font-weight: semi-bold;
+            // ${props => props.theme.textColor};
+        }
+        h1{
+            color: black;
```
Comment (Tanuj22) [inline src/sections/SMI/smi.style.js:13]: Redundant

File: src/sections/SMI/smi.style.js [hunk 8]
```diff
@@ -0,0 +1,146 @@
+import styled from "styled-components";
+
+const SMIWrapper = styled.section`
+    .description{
+        margin: 6.25rem -0.94rem 7.5rem -0.94rem;
+        h6{
+            font-size: 20px;
+            color: #1E2117;
+            font-weight: semi-bold;
+            // ${props => props.theme.textColor};
```
Comment (Tanuj22) [inline src/sections/SMI/smi.style.js:10]: Remove

File: src/sections/SMI/smi.style.js [hunk 9]
```diff
@@ -0,0 +1,146 @@
+import styled from "styled-components";
+
+const SMIWrapper = styled.section`
+    .description{
+        margin: 6.25rem -0.94rem 7.5rem -0.94rem;
+        h6{
+            font-size: 20px;
+            color: #1E2117;
```
Comment (Tanuj22) [inline src/sections/SMI/smi.style.js:8]: Add this color to global them if not already present

File: src/sections/SMI/smi.style.js [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+            font-weight: semi-bold;
+            // ${props => props.theme.textColor};
+        }
+        h1{
+            color: black;
+            margin-bottom: 1.56rem;
+        }
+        h5{
+            color: #1E2117;
```
Comment (Tanuj22) [inline src/sections/SMI/smi.style.js:17]: Same

File: src/sections/SMI/smi.style.js [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+    }
+    .hero-img{
+        margin: -6.25rem 0 0 0;
+        object-fit: none;
+        width: 36rem;
+    }
+    .feature-table{
+        margin: 0 5rem 1.875rem 5rem;
+    }
+    .pricing-block{
```
Comment (Tanuj22) [inline src/sections/SMI/smi.style.js:48]: Better name

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
The image should be cut at the side like in the design 
Screenshot from 2021-01-05 21-35-32

File: src/sections/SMI/smi.style.js (hunk 9 above)
Comment (Nikhil-Ladha) [reply src/sections/SMI/smi.style.js:8]: I think this is the jungle green color and it's already there in the theme.

File: src/sections/SMI/smi.style.js (hunk 7 above)
Comment (Jashpatel1) [reply src/sections/SMI/smi.style.js:13]: h1 is given dark jungle green, but the design demands black.

Comment (Nikhil-Ladha) [reply src/sections/SMI/smi.style.js:13]: None of the designs have the header assigned the color jungle green. So, that will not be an issue.

Comment (Nikhil-Ladha) [reply src/sections/SMI/smi.style.js:13]: Moreover, it won't look bad and much different if we use jungle green....but the good part will be that we will adhere to the brand colors.

**Comment by Tanuj22** [comment]
This text is not at the center 
Screenshot from 2021-01-06 22-08-05

Also, add a project card for this

**Comment by Jashpatel1** [comment]
Apart from the hero image, this PR is complete. I will have a look at some way to get it done. Please have a look @leecalcote @Nikhil-Ladha @Tanuj22

**Comment by Tanuj22** [comment]
The button is getting overlapped with the hero img for smaller screens. Fix that

**Comment by Jashpatel1** [comment]
> The button is getting overlapped with the hero img for smaller screens. Fix that

Done

**Comment by Tanuj22** [comment]
Use the image having three starting logos as per the design

**Comment by Tanuj22** [comment]
@Jashpatel1 there is a conflict

**Review by Tanuj22 — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
Yay! 😄

**Pull request merged**